sql/integration.spec.ts

```diff
@@ -13,7 +13,7 @@ suite("builders", { parallel: true }, ({ before, after }) => {
 
   before("create bigquery", async () => {
     bigquery = await dbadapters.create(
-      dfapi.credentials.read("bigquery", "test_credentials/bigquery.json"),
+      { ...dfapi.credentials.read("bigquery", "test_credentials/bigquery.json"), location: "EU" },
```

Comment:
This makes it more accurate to what we do in the web app. Previously we stored a location in the credentials, whereas what we actually do in the app is have people upload credentials (often with no location), then set the location or override it in the UI. This approach makes sense because it makes it easier to change location.

Reply:
I don't think this is true? The location setting in the UI maps into the credentials object. https://github.com/dataform-co/dataform/blob/master/protos/profiles.proto#L44

For clarity, there are 2 types of credentials here:

- BigQuery service account credentials (emails, client secrets, billing project-id, no location info)
- Dataform BigQuery credentials (BQ service account credentials + location and project-id)